### PR TITLE
migrated: pages/developers/learning-tools.tsx to Chakra UI

### PR DESCRIPTION
**Pull request by Seek4samurai**
## Description
Migrated pages/developers/learning-tools.tsx to Chakra UI.

## Related Issue
https://github.com/ethereum/ethereum-org-website/issues/9353

### CODE REVIEW

File: src/pages/developers/learning-tools.tsx [hunk 1]
```diff
@@ -1,78 +1,103 @@
...
+  </Subtitle>
+)
+
+const ContentBox = (props: ChildOnlyProp) => (
+  <Box pt={4} pb={4} pl={8} pr={8} w={"100%"} {...props}></Box>
+)
+
+const StackContainer = (props: ChildOnlyProp) => (
+  <Box
+    border="1px solid border"
```
Comment (minimalsm) [inline src/pages/developers/learning-tools.tsx:86]:
```suggestion
    border="1px solid"
    borderColor="border"
```

File: src/pages/developers/learning-tools.tsx [hunk 2]
```diff
@@ -1,78 +1,103 @@
...
-  font-size: 1.25rem;
-  line-height: 1.4;
-  font-weight: 400;
-  color: ${(props) => props.theme.colors.text300};
-  max-width: 55ch;
-  margin-bottom: 0.5rem;
-  margin-top: 1rem;
-`
+const H1 = (props: ChildOnlyProp) => (
+  <Flex
```
Comment (minimalsm) [inline src/pages/developers/learning-tools.tsx:47]: This should be an H1

```suggestion
  <Heading
    as="h1"
```

File: src/pages/developers/learning-tools.tsx [hunk 3]
```diff
@@ -1,78 +1,103 @@
...
+    m={"8 0 0"}
+    mt={0}
+    color="text"
+    fontStyle="normal"
+    fontFamily={"mono"}
+    textTransform="uppercase"
+    fontWeight="semibold"
+    fontSize="4xl"
```
Comment (minimalsm) [inline src/pages/developers/learning-tools.tsx:55]: Let's stick to current design, which is 32px.

```suggestion
    fontSize="2rem"
```

File: src/pages/developers/learning-tools.tsx [hunk 4]
```diff
@@ -1,78 +1,103 @@
...
-    margin-left: 0rem;
-    margin-right: 0rem;
-    border-radius: 0px;
-  }
-`
+const SubtitleTwo = (props: ChildOnlyProp) => (
+  <Subtitle>
+    <Box mt={0} {...props}></Box>
+  </Subtitle>
+)
```
Comment (minimalsm) [inline src/pages/developers/learning-tools.tsx:78]: Currently, this doesn't remove the margin on the Subtitle (which is what we want). It adds another element inside of it, but the parent keeps its margin-top.

We can do this:

```suggestion
const SubtitleTwo = (props) => (
  <Subtitle mt={0} {...props}/>
)
```

File: src/pages/developers/learning-tools.tsx [hunk 5]
```diff
@@ -1,78 +1,103 @@
...
+  />
+)
 
-const SubtitleTwo = styled(Subtitle)`
-  margin-top: 0rem;
-`
+const Subtitle = (props: ChildOnlyProp) => (
+  <Flex
+    fontSize={"xl"}
+    lineHeight={"short"}
```
Comment (minimalsm) [inline src/pages/developers/learning-tools.tsx:65]: Just noting, this is a tiny bit different from what we have currently. Not sure if we care about this @pettinarip?

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @Seek4samurai, nice work on this 💪

I've Made some suggested tweaks to get this closer to the current design. 

The main thing we need to solve is that we are setting some breakpoints the wrong way round (e.g. 0px margin on big screens and 8px on small, instead of the opposite). I've left suggestions that I believe fixes all of these 🫡

**Review by minimalsm — DISMISSED** [review]
Hey @Seek4samurai, nice work on this 💪

I've Made some suggested tweaks to get this closer to the current design. 

The main thing we need to solve is that we are setting some breakpoints the wrong way round (e.g. 0px margin on big screens and 8px on small, instead of the opposite). I've left suggestions that I believe fixes all of these 🫡

**Comment by Seek4samurai** [comment]
Ok maybe I did messed up some breakpoints. Thanks!

File: src/pages/developers/learning-tools.tsx (hunk 5 above)
Comment (pettinarip) [reply src/pages/developers/learning-tools.tsx:65]: Yea, we want to keep the current styles so this should be `lineHeight={1.4}`

**Review by pettinarip — APPROVED** [review]
Thanks for the PR @Seek4samurai.

I've applied the suggestions from the reviews. This is good to go now 💪🏼

**Pull request merged**